From c9e09b6bf29000a5df64b7154a2eb0da8e394eb4 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair@alistair23.me>
Date: Mon, 13 Apr 2020 19:00:38 -0700
Subject: [PATCH] arm64: allwinner: sopine-baseboard: Expose LSM9DS1 IMU

Signed-off-by: Alistair Francis <alistair@alistair23.me>
---
 .../allwinner/sun50i-a64-sopine-baseboard.dts | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts
index 5b825e628df2..9ea25c57929e 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts
@@ -164,6 +164,28 @@ gt911@5d {
 	};
 };
 
+&i2c1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&i2c1_pins>;
+	status = "okay";
+
+	lsm9ds1-imu@6b {
+		compatible = "st,lsm9ds1-imu";
+		reg = <0x6b>;
+
+		interrupt-parent = <&pio>;
+		interrupts = <7 9 IRQ_TYPE_EDGE_RISING>; /* PH9 */
+	};
+
+	lsm9ds1-magn@1e {
+		compatible = "st,lsm9ds1-magn";
+		reg = <0x1e>;
+
+		interrupt-parent = <&pio>;
+		interrupts = <2 7 IRQ_TYPE_EDGE_RISING>; /* PC7 */
+        };
+};
+
 &reg_ldo_io0 {
 	regulator-min-microvolt = <3300000>;
 	regulator-max-microvolt = <3300000>;
-- 
2.26.0

