From 2f047c0f8bc11a59a3b800b9a5ea52725eaa88c1 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair@alistair23.me>
Date: Tue, 21 Apr 2020 23:17:53 -0700
Subject: [PATCH] arm64: allwinner: sopine-baseboard: Expose MLX90614 temp

Signed-off-by: Alistair Francis <alistair@alistair23.me>
---
 .../dts/allwinner/sun50i-a64-sopine-baseboard.dts    | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts
index 8d79b6a141ae..7d1dfd986691 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts
@@ -195,6 +195,18 @@ gt911@5d {
 	};
 };
 
+&i2c1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&i2c1_pins>;
+	status = "okay";
+
+	mlx90614@5a {
+		compatible = "melexis,mlx90614";
+		reg = <0x5a>;
+		wakeup-gpios = <&pio 7 3 GPIO_ACTIVE_HIGH>; /* PH3 */
+	};
+};
+
 &reg_ldo_io0 {
 	regulator-min-microvolt = <3300000>;
 	regulator-max-microvolt = <3300000>;
